python/core/auto_generated/geometry/qgsabstractgeometry.sip.in

```diff
@@ -719,6 +719,22 @@ Converts the geometry to a specified type.
 :return: ``True`` if conversion was successful
 
 .. versionadded:: 2.14
+%End
+
+    virtual const QgsAbstractGeometry *simplifiedTypeRef() const /HoldGIL/;
+%Docstring
+Returns a reference to the simplest lossless representation of this geometry,
+e.g. if the geometry is a multipart geometry type with a single member geometry,
+a reference to that part will be returned.
+
+This method employs the following logic:
+
+- For multipart geometries containing a single part only a direct reference to that part will be returned.
+- For compound curve geometries containing a single curve only a direct reference to that curve will be returned.
+
+This method returns a reference only, and does not involve any geometry cloning.
```

Comment:
Should we add something like the following to be in line with our standard taxonomy.

> The ownership is not transferred.

Reply:
Done!